src/resources/js/frontend-ticket-form.js

```diff
@@ -0,0 +1,1779 @@
+// @TODO: Take this line off once we actually have the tribe object
```

Comment:
Can you confirm we can delete this JS file from the PR? I know we deprecated the code that called this and it now just includes the other JS file instead of this one.

Reply:
I think there might be an argument for keeping it in place in case we get someone running an older version of ETP?

Note similar conversation with @jesseeproductions here: https://github.com/moderntribe/event-tickets/pull/1547#discussion_r367006467

Reply:
The JS file was removed and we called the tickets-block.js in the deprecated method.